Gemfile.lock

```diff
@@ -292,9 +290,9 @@ GEM
     selenium-webdriver (3.142.7)
       childprocess (>= 0.5, < 4.0)
       rubyzip (>= 1.2.2)
-    simple_form (5.0.3)
-      actionpack (>= 5.0)
-      activemodel (>= 5.0)
+    simple_form (5.1.0)
```

Comment:
### [simple_form](https://github.com/heartcombo/simple_form)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 5.0.3 → 5.1.0

[_~~change-log~~, [source-code](https://github.com/plataformatec/simple_form)_]





<details>
<summary>Commits</summary>

A change of **11** commits. See the full changes on [the compare page](https://github.com/heartcombo/simple_form/compare/v5.0.3...v5.1.0).

These are the first 10 commits:
- (b8b6841) [Remove now unused sudo option for travis](https://github.com/heartcombo/simple_form/commit/b8b6841c8a4c16fbce9b54c028c01f18f2c43922)
- (0186ada) [Enable email notifications for now, remove slack config](https://github.com/heartcombo/simple_form/commit/0186adacc54b9373eaeb3a8fa977b8216fb30259)
- (689f5e6) [Drop support to Ruby < 2.4, update travis to run on latest v…](https://github.com/heartcombo/simple_form/commit/689f5e63a8a688001427e07934ace3e14131daf2)
- (4312bba) [Remove unused methods](https://github.com/heartcombo/simple_form/commit/4312bba6525f1ad65010c7c244e039e6d8f9b7a5)
- (4f829d6) [Merge pull request #1719 from jonesdeini/master](https://github.com/heartcombo/simple_form/commit/4f829d6966abc6c0d43c784e052e9e695fd4ccdd)
- (f0b4123) [Move from travis to GitHub Actions](https://github.com/heartcombo/simple_form/commit/f0b412301b10e64a835cd6e711a4b6be802002d6)
- (e0df150) [Rename gemfiles](https://github.com/heartcombo/simple_form/commit/e0df1502fa90f3259f3250562e42d9a8d11e78f2)
- (19e7394) [Update to Rails 6.1, drop support to Rails < 5.2](https://github.com/heartcombo/simple_form/commit/19e739486c8592f4e468234b09ae629060abdfba)
- (0595088) [Add support for Ruby 3, drop support for Ruby < 2.5, remove …](https://github.com/heartcombo/simple_form/commit/05950880ba8750472a7fd62aba9ee79f49f530f6)
- (cbcfe2e) [Add note about Ruby/Rails support versions in the Readme](https://github.com/heartcombo/simple_form/commit/cbcfe2e972fc25a53474d72b256a7f6c389de545)


</details>